src/main/java/hudson/plugins/clover/CloverBuildAction.java

```diff
@@ -149,10 +156,20 @@ static CloverBuildAction getPreviousResult(Run<?, ?> start) {
         if (c != null) {
             c.setOwner(build);
         }
+        if( this.projectActions == null )
```

Comment:
This is due to serialization.
I noticed that in case of Jenkins restart constructor is not called any more and projectAction field is null.